examples/favorites_advanced/redux_favorites_advanced/lib/main.dart

```diff
@@ -0,0 +1,28 @@
+import 'package:flutter/material.dart';
+import 'package:redux/redux.dart';
+
+import 'feature_home/models/navigation_state.dart';
+import 'feature_home/redux/reducers.dart';
+
+import 'feature_home/views/home_page.dart';
+
+void main() {
+  runApp(MyApp());
+}
+
+class MyApp extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    final store = Store<NavigationState>(
```

Comment:
@GoranPrime, what are the best practices in redux regarding store creation? Where is the best place to create the store? In the page/widget, where your StoreProvider resides (in our case `HomePage`), or in the parent widget ( in our case is `MyApp`) ?

Reply:
It is recommended to create the store in `main()` and wrap the `MaterialApp` in a `StoreProvider`, so I've changed it to that.